gobblin-hive-registration/src/main/java/gobblin/hive/spec/HiveSpec.java

```diff
@@ -33,25 +33,15 @@
   public Path getPath();
 
   /**
-   * Get the Hive database name.
+   * Get the Hive {@link Table}.
    */
-  public String getDbName();
+  public Table getTable();
```

Comment:
Can you add more doc here indicating that `Table` returned here is not registered yet?